R/duplicate_argument_linter.R

```diff
@@ -1,6 +1,9 @@
 #' Duplicate argument linter
 #'
-#' Check for duplicate arguments in function calls.
+#' Check for duplicate arguments in function calls. Some of these cases will be
+#' blocked by R's syntax itself (e.g. `mean(x = 1:5, x = 2:3)`), but even when
+#' such usage *is* allowed, it is rarely desirable and should be used sparingly
```

Comment:
Not a fan of this wording, given the utility of usage in `mutate()` to sequentially apply transformations to a single column. How about

Some cases are run-time errors (e.g. `mean(x = 1:5, x = 2:3)`), otherwise this linter is used to discourage explicitly providing duplicate names to objects (e.g. `c(a = 1, a = 2)`). Duplicate-named objects are hard to work with programmatically and should typically be avoided.

Reply:
Good point. I have reverted this whole change.

Reply:
And accepted the suggested text instead.